index.html

```diff
@@ -10,44 +10,231 @@
                     </div>
+                    <p>the quick fox jumps over the lazy dog the quick fox jumps over the lazy dog</p>
+                </div>
+
+                <div class="card-header">
+                    <div class="card__content">
+                    <img src="img/people.svg" alt="icon_people">
+                    <h5>National top 50 firms</h5>
+                    </div>
+                    <p>the quick fox jumps over the lazy dog the quick fox jumps over the lazy dog</p>
+                </div>
+
+                <div class="card-header">
+                    <div class="card__content">
+                    <img src="img/graph.svg" alt="icon_graph">
+                    <h5>National top 50 firms</h5>
+                    </div>
+                    <p>the quick fox jumps over the lazy dog the quick fox jumps over the lazy dog</p>
+                </div>
+
+                <div class="card-header">
+                    <div class="card__content">
+                    <img src="img/medal.svg" alt="icon_medal">
+                    <h5>Web Development</h5>
 
-                    <div class="navbar">
-                        <nav>
-                            <ul class="right-nav">
-                                <li><a href="#">Home</a></li>
-                                <li><a href="#">Product</a></li>
-                                <li><a href="#">Pricing</a></li>
-                                <li><a href="#">Contact</a></li>
-                            </ul>
-                        </nav>
+                    </div>
+                    <p>the quick fox jumps over the lazy dog the quick fox jumps over the lazy dog</p>
+                </div>
+            </div>
+        </section>
 
-                        <ul class="right-nav-icons">
+
+
+
+        <section class="footer">
+            <div class="footer-header">
+                <div class="logo-box">
+                    <a href="#" class="logo"><img src="img/KMarketer.svg" alt="logo"></a>
+                </div>
+                
+                <nav>
+                    <ul class="footer-media">
+                        <li>
+                            <a href="#"><img src="img/icon_facebook.svg" alt="Facebook"></a>
+                        </li>
+                        <li>
+                            <a href="#"><img src="img/icon_instagram.svg" alt="Instagram"></a>
+                        </li>
+                        <li>
+                            <a href="#"><img src="img/icon_twitter.svg" alt="Twitter"></a>
+                        </li>
+                        <li>
+                            <a href="#"><img src="img/icon_youtube.svg" alt="Youtube"></a>
+                        </li>
+                    </ul>
+                </nav>
+            </div>
+
+            <hr>
+
+            <div class="footer-container">
+                <div class="footer-columns">
+                    <div class="scan-columns-align">
```

Comment:
I don't really get the meaning of "scan-columns-align" class name.

Reply:
changed names' classes